### ISSUE TITLE
IANA registry rules

### ISSUE THREAD

**Issue by martinthomson** [tlswg/tls-flags#11]
My interpretation of these is that these reduce to something like:

* 0-31 - standards action
* 32-63 - ??? (something like RFC Required or maybe Specification Required)
* 64-79 - might as well burn these for all the good they will do
* 80+ - open season (FCFS?)

I can see the rationale for standards action here, but the rest of this should just be expert review (or specification required) along with guidance to experts.

As this requires judgment, and the proponents of a new flag will lack the necessary perspective to have good judgment, a lot does depend on the nature of the guidance to experts.

The document will need to provide experts with support that they can rely on: right now, the text does not do that.  It relies too heavily on good judgment.  If there is concrete advice that can be given about how to pick a value, then the document should do that.  If it can't, as I suspect is the case, then it should either give experts the power to pick or just let authors grab for short values.

**Comment by yoavnir** [comment]
All TLS registries are "expert review" with no FCFS. I don't think we want this one to be the outlier.

So how about making the segmentation in section 4, something like
```
The values is the registry will be assigned as follows:
 * Values 0-31 shall be Standards Action, with the value assigned at the discretion of the IANA expert.
 * Values 32-64 shall be Specification Required and assigned sequentially
 * Values 64-80 shall be reserved for private use
 * Values 80 and above shall be Specification Required and assigned sequentially after the range 32-64 has been exhausted.
```

Then I can reduce the content of section 4.1 and just advise the expert to assign the first 7 bits only with a specific request from the WG. I don't want to specify the **TLS** working group, because something like this could potentially (though not likely) come from UTA or some such.

**Comment by martinthomson** [comment]
I don't think that sequential assignment has worked well for us.  I would remove that part and maybe change the assignment rule to be "at the discretion of the expert" across the board, with a reminder that small values are more valuable and so should be reserved for things that are going to be used more often.

**Comment by chris-wood** [comment]
Fixed by #14.
